### ISSUE TITLE
No audible feedback for buttons inside the toolbar

### ISSUE THREAD

**Issue by dariiing** [cryptpad/cryptpad#1792]
The buttons from the toolbar are not correctly announced, there should be clear description for each button.
Currently the screen reader reads the buttons as "Button" without giving more context.
